libs/dh/wholesale/feature-calculations/src/lib/details/details.component.html

```diff
@@ -71,10 +71,12 @@ <h3 class="headline watt-headline-2">{{ t("calculationTypes." + type()) }}</h3>
         [label]="t('details.executionTime')"
         [value]="startedAtOrScheduledAt() | wattDate: 'long' | dhEmDashFallback"
       />
-      <watt-description-list-item
-        [label]="t('details.period')"
-        [value]="calculation?.period | wattDate: 'short' | dhEmDashFallback"
-      />
+      @if (wholesaleAndEnergyCalculation?.period || query.loading()) {
```

Comment:
question: is the `query.loading()` intentional here? None of the other fields have it.

Reply:
it is intentional, yes 🙂 it sticks out I agree, but it is because `period` makes no sense for electrical heating calculations, so in that case they want to just hide the field entirely. but since I didn't want the field to "pop in" for normal calculations, I included the loading check, so it is only hidden once it knows for sure if it is an electrical heating calculation or not.

but I can see if I can make the logic a bit clearer on that

Reply:
I see. Well, if it's intentional then it's ok to leave as is. I thought it looked like a leftover from testing something, that's why I pointed it out.